server/Arcadia.Assistant.CSP/Vacations/CspVacationApprovalsChecker.cs

```diff
@@ -37,16 +37,17 @@ protected override async Task<string> GetNextApprover(string employeeId, IEnumer
             var parentDepartment = allDepartments.FirstOrDefault(d => d.DepartmentId == employeeDepartment.ParentDepartmentId);
             var isEmployeeChief = employeeDepartment.ChiefId == employeeId;
 
-            var neededApprovals = new List<string>();
+            string mainApproverId = null;
+            string secondApproverId = null;
```

Comment:
so there won't be three approvers? :) what you can have is finalApprover / preapprovers

Reply:
I made it as simple as possible, having in mind that it is temporary
solution anyway.